packages/contentful--app-scripts/src/create-app-definition/build-app-definition-settings.ts

```diff
@@ -87,14 +94,61 @@ NOTE: This will create an app definition in your Contentful organization.
         return true;
       },
     },
+    {
+      name: 'pageNav',
+      message: 'Page location: Would you like your page location to render in the main navigation?',
+      type: 'confirm',
+      default: false,
+      when(answers) {
+        return answers.locations.includes('page');
+      },
+    },
+    {
+      name: 'pageNavLinkName',
+      message: 'Page location: Provide a name for the link in the main navigation:',
+      when(answers) {
+        if (answers.locations.includes('page') && answers.pageNav) {
+          return answers.name;
```

Comment:
I see this is working great as is but was thinking we might want:

```suggestion
          return !!answers.name;
```

just in case inquirer ever starts enforcing that the return value is really a boolean rather than just truthy